src/components/Quiz/QuizWidget.tsx

```diff
@@ -301,7 +301,7 @@ const QuizWidget: React.FC<IProps> = ({ quizKey, maxQuestions }) => {
           size="50px"
           bg={
             !showAnswer
-              ? "primary"
+              ? "primary.base"
               : currentQuestionAnswerChoice?.isCorrect
               ? "success"
               : "error"
```

Comment:
![image](https://github.com/ethereum/ethereum-org-website/assets/468158/77636b5e-ec68-4fad-9103-b1b12970e2ae)

```suggestion
              ? "success.base"
              : "error.base"
```

Reply:
@pettinarip note that for the second item you point out in the screenshot ("Sidechains") this is unrelated as the rendering of `white` is coming from logic for [QuizRadioGroup](https://github.com/ethereum/ethereum-org-website/blob/279659c5fafb754900c04bedc180402900c3d026/src/components/Quiz/QuizRadioGroup.tsx#L100-L104). 

This indeed looks like an error in color choice in light mode!

Reply:
I think the `white` color is correct. The thing missing here is the red background.

This is in [dev](https://ethereumorgwebsitedev01.gatsbyjs.io/en/what-is-ethereum/)
![image](https://github.com/ethereum/ethereum-org-website/assets/468158/b2e6b503-a9a4-42a7-a623-5c5a260b4283)

Reply:
Ahh found it! Pushed the fix.